tests/integration/v2/services.go

```diff
@@ -110,7 +110,7 @@ func GasMeterFactory(ctx context.Context) func() gas.Meter {
 }
 
 func (s storeService) OpenKVStore(ctx context.Context) corestore.KVStore {
-	const gasLimit = 100_000
+	const gasLimit = 1_000_000
```

Comment:
why the increase?

Reply:
Few of deterministic tests are failing with out of gas issue.